conan/packager.py

```diff
@@ -67,13 +67,22 @@ def __init__(self, args=None, username=None, channel=None, runner=None,
         self.upload_retry = upload_retry or os.getenv("CONAN_UPLOAD_RETRY", 3)
         self.reference = reference or os.getenv("CONAN_REFERENCE", None)
         self.password = password or os.getenv("CONAN_PASSWORD", None)
-        self.remote = remote or os.getenv("CONAN_REMOTE", None)
-        self.upload = upload or (os.getenv("CONAN_UPLOAD", None) in ["True", "true", "1"])
+        self.remotes = remotes or os.getenv("CONAN_REMOTES", None)
+        self.upload = upload or os.getenv("CONAN_UPLOAD", None)
         self.stable_branch_pattern = stable_branch_pattern or os.getenv("CONAN_STABLE_BRANCH_PATTERN", None)
         default_channel = channel or os.getenv("CONAN_CHANNEL", "testing")
         stable_channel = stable_channel or os.getenv("CONAN_STABLE_CHANNEL", "stable")
         self.channel = self._get_channel(default_channel, stable_channel)
 
+        if upload:
+            if upload in ("0", "None", "False"):
+                self.upload = None
+            elif upload == "1":
```

Comment:
The builds will break anyway. It is necessary to specify the bintray remote to upload to, that is not handled by conan-package-tools now.